cpp/src/copying/contiguous_split.cu

```diff
@@ -151,11 +391,11 @@ struct column_copy_functor {
                                validity, validity == nullptr ? UNKNOWN_NULL_COUNT : 0 };      
       if(in.nullable()){               
          copy_in_place_kernel<block_size, T, true><<<grid.num_blocks, block_size, 0, 0>>>(
-                           *column_device_view::create(in_wrapped), 
+                           *column_device_view::create(in_wrapped),                            
```

Comment:
The change here and in the same line in the else looks like it is just adding spaces to the end of the line.

Reply:
Yeah, sorry.  This is a weird artefact of the way I highlight/delete lines.